models/bridge/bridges_schema.yml

```diff
@@ -0,0 +1,117 @@
+version: 2
+
+models:
+  - name: bridge_flows
+    meta:
+      blockchain: ethereum, optimism
+      sector: bridge
+      contributors: msilb7, soispoke
+    config:
+      tags: ['ethereum', 'optimism', 'bridge', 'flows', 'cross-chain']
+    description: >
+      Cross-chain bridge transfers on all chains across all contracts and versions
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+              - block_date
+              - blockchain
+              - tx_hash
+              - evt_index
+              - transfer_id
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain Used to Pull Data - What chain the bridge transfers are in prespective from"
+      - &project
+        name: project
+        description: "Bridge protocol project"
+      - &version
+        name: version
+        description: "Bridge protocol project version"
+      - &block_time
+        name: block_time
+        description: "Block time of the transaction"
+      - &block_date
+        name: block_date
+        description: "Date of the block time"
+      - &block_number
+        name: block_number
+        description: Block number of the transaction
+      - &tx_hash
+        name: tx_hash
+        description: Transaction hash
+      - &bridge_path_name
+        name: bridge_path_name
+        description: "Bridge Path: Source -> Destination by Chain name"
+      - &sender
+        name: sender
+        description: "Address of the wallet sending the transfer"
+      - &receiver
+        name: receiver
+        description: "Address of the wallet receiving the transfer"
+      - &transfer_type
+        name: transfer_type
+        description: "Is the transfer a Deposit, or a Withdrawal away?"
+      - &token_symbol
+        name: token_symbol
+        description: Bridged Token Symbol
+      - &token_amount
+        name: oken_amount
+        description: Bridged Token Amount (Decimal adjusted)
+      - &token_amount_usd
+        name: token_amount_usd
+        description: Bridged Token Amount USD
+      - &token_amount_native
+        name: token_amount_usd
+        description: Bridged Token Amount ETH
+      - &token_amount_raw
+        name: token_amount_raw
+        description: Bridged Token Amount (Not Decimal adjusted)
+      - &token_fee_amount
+        name: token_fee_amount
```

Comment:
Good point to capture fees, and from what I see here, it's possible to have fees paid in different asset than the bridged token, correct? (this would make sense).

I would probably adjust the naming a bit, since `token_fee_...` seems a bit confusing to me.

Í'd go:

<html>
<body>
<!--StartFragment--><b style="font-weight:normal;" id="docs-internal-guid-7c602076-7fff-35cd-4d3e-eb591087acd2"><div dir="ltr" style="margin-left:0pt;" align="left">

Column name | Column description | dtype | Soft / Hard requirements* | Example
-- | -- | -- | -- | --
fee_amount | fee amount paid denominated in the asset used | numeric | not null | 21.2
fee_amount_usd | fee amount denominated in USD | numeric | not null | 22.7
fee_amount_raw | ... | numeric | not null | 21233124100014230
fee_token_symbol | asset used to pay the bridge fee | string | not null | USDC
fee_token_address | contract address of the asset used to pay the fee | string | not null | 0x..

</div></b><!--EndFragment-->
</body>
</html>